DEPRECATIONS.md

```diff
@@ -10,6 +10,40 @@ Old versions of AL-Go for GitHub uses old and unsupported versions of GitHub act
 
 When handling support requests, we will request that you to use the latest version of AL-Go for GitHub and in general, fixes will only be made available in a preview version of AL-Go for GitHub and subsequently in the next version released.
 
+## Changes in effect after October 1st 2025
+
+<a id="alwaysBuildAllProjects"></a>
+
+### Setting `alwaysBuildAllProjects` will no longer be supported
+
+The old setting would determine whether or not to build all project during the Pull Request workflow. With incremental Builds now supported, please set the `onPull_Request` property of the `incrementalBuilds` setting to false to force full builds in Pull Requests.
+
+```
+"incrementalBuilds": {
+    "onPull_Request": false
+}
+```
+
+<a id="_workflow_Schedule"></a>
+
+### Setting `<workflow>Schedule` will no longer be supported
+
+The old setting, where the setting key was a combination of the workflow name and `Schedule` (dynamic setting key name) is no longer supported. Instead you need to use a setting called [workflowSchedule](https://aka.ms/algosettings#workflowSchedule) and either use [Conditional Settings](https://aka.ms/algosettings#conditional-settings) or place the setting in a workflow specific settings file. Example using conditional settings:
+
+```
+"conditionalSettings": [
+    {
+        "workflows": [ "Update AL-Go System Files" ],
+        "settings": {
+            "workflowSchedule": "30 0 * * 0"
```

Comment:
Can we have this setting as a structure.
```
"workflowSchedule": {
    cron: "30 0 * * 0"
```
Or something like that?
In #1037 I'd like to add `includeBranches`:
```
"workflowSchedule": {
    "cron": "30 0 * * 0",
    "includeBranches": [ "main", "releases/*"]
```
for multi-branch workflow runs.

Reply:
Sounds like a good idea - it would also allow us to use the same mechanism for adding parameters to f.ex. Publish To Environment when running on a schedule - although only allowing for one schedule and one set of parameters.